### PR TITLE
feat(security): Remove Writable from security services

### PR DESCRIPTION
**Pull request by jim-wang-intel**
Update for security services on:

- Loglevel is on the top level of ConfigurationStruct
- Remove Writable from Configuration struct
- Make UpdateFromRaw, UpdateWritableFromRaw do nothing and make EmptyWritablePtr just return nil

Fixes: #3123

Signed-off-by: Jim Wang <yutsung.jim.wang@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/edgex-go/blob/master/.github/Contributing.md.

## What is the current behavior?

security services have Writable struct but actually it never uses

## What is the new behavior?
Refactor the `LogLevel` inside Writable struct and remove Writable.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x ] No

## New Imports

- [ ] Yes
- [x ] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?
Depends on PR #2863 as the redis' toml conf would also be removed about the Writable section.

## Other information

### CODE REVIEW

File: cmd/security-proxy-setup/res/configuration.toml [hunk 1]
```diff
@@ -16,8 +16,9 @@
 
 # This is a TOML config file for edgexsecurity service.
 
-[Writable]
 LogLevel = "DEBUG"
+
+[ProxySetup]
```
Comment (lenny-intel) [inline cmd/security-proxy-setup/res/configuration.toml:21]: Why was this added?

File: internal/security/proxy/config/config.go [hunk 2]
```diff
@@ -33,8 +35,7 @@ type ConfigurationStruct struct {
 	Clients       map[string]bootstrapConfig.ClientInfo
 }
 
-type WritableInfo struct {
-	LogLevel       string
+type ProxySetupInfo struct {
 	RequestTimeout int
```
Comment (lenny-intel) [inline internal/security/proxy/config/config.go:39]: Why does this need to be in a sub struct? Just have it add the top level also.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: cmd/security-proxy-setup/res/configuration.toml (hunk 1 above)
Comment (jim-wang-intel) [reply cmd/security-proxy-setup/res/configuration.toml:21]: no particular strong reason, i just felt that it can be more structural in this way since potentially you can have many more `RequestTimout` for that naming alone.

Comment (lenny-intel) [reply cmd/security-proxy-setup/res/configuration.toml:21]: It is redundant since this is `Proxy Setup's` specific configuration. IMO, the extra level isn't needed. We added the extra levels in the bootstrapper config to make the Env override more readable, but that isn't needed here.

Comment (jim-wang-intel) [reply cmd/security-proxy-setup/res/configuration.toml]: fixed.

File: internal/security/proxy/config/config.go (hunk 2 above)
Comment (jim-wang-intel) [reply internal/security/proxy/config/config.go:29]: fixed.

**Review by lenny-intel — DISMISSED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]
